src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -976,3 +981,52 @@ export const handleGetMetadata = [
   }),
   getMetadata,
 ] as ControllerHandler[]
+
+const getS3PresignedPostData: ControllerHandler<
```

Comment:
reminder to add some jsdocs and test

Reply:
docs added here: [a42c52e](https://github.com/opengovsg/FormSG/pull/6685/commits/a42c52e7157af3dd0dded81a2574710fd328eb76)

Reply:
I've added the tests for the `/submissions/storage/get-s3-presigned-post-data` route in [6922dce](https://github.com/opengovsg/FormSG/pull/6685/commits/6922dceba9ff631b87946ff8c0443beb08521f0c). Is it okay if I skip this set of controller tests and focus on adding tests for `/submissions/storage` instead?

Reply:
thanks for doing up the route tests! haha ok i chip in a bit here bah just for completeness 😅  added the controller unit tests in [ffd798b](https://github.com/opengovsg/FormSG/pull/6685/commits/ffd798b9be0d3b11af03028c274c57cae2657b56) for completeness.

Reply:
should be good to go once you add in `/submissions/storage` tests

Reply:
thanks so much @tshuli! 🙏 this PR is much bigger than I expected 😓